mx.truffleruby/graalvm.properties

```diff
@@ -0,0 +1,20 @@
+# This file contains native-image arguments needed to build truffleruby
```

Comment:
Why is this file separate and native-image.properties is not used?
For `-Dorg.graalvm.launcher.standalone` ? cc @pitr-ch

Reply:
`standalone` is rather to alter the behavior of Launcher to work outside of GraalVM. I have nothing against removing it, there just has to other support for standalone launcher in place.